invenio_rdm_records/services/access/service.py

```diff
@@ -71,6 +71,11 @@ def schema_grant(self):
         """Schema for secret links."""
         return ServiceSchemaWrapper(self, schema=self.config.schema_grant)
 
+    @property
+    def schema_parent(self):
+        """Schema for record parent."""
+        return ServiceSchemaWrapper(self, schema=self.config.schema_parent)
```

Comment:
we should not return the parent but rather the parent settings...shall we rename this schema to `schema_parent_access_settings` and be super explicit of what do we return?